include/eve/module/math/regular/impl/pow_abs.hpp

```diff
@@ -16,7 +16,8 @@ namespace eve::detail
 {
 template<floating_real_value T, floating_real_value U>
 EVE_FORCEINLINE auto
-pow_abs_(EVE_SUPPORTS(cpu_), T const& a, U const& b) noexcept requires compatible_values<T, U>
+pow_abs_(EVE_SUPPORTS(cpu_), T const& a, U const& b) noexcept
+//-> decltype(pow(eve::abs(a), b))
```

Comment:
why the comment ?

Reply:
does not compile and anyway seems unnecessary

Reply:
becasue it has to be eve::pow I guess